platform/wayland/cog-view-wl.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * cog-view-wl.h
+ * Copyright (C) 2023 Pablo Saavedra <psaavedra@igalia.com>
+ * Copyright (C) 2023 Adrian Perez de Castro <aperez@igalia.com>
+ *
+ * Distributed under terms of the MIT license.
```

Comment:
While at it, let's use SPDX license identifiers for new files, so there will be less to change when we tackle #357 😉 

```suggestion
 * SPDX-License-Identifier: MIT
```

Reply:
applied